src/machine-config.h

```diff
@@ -112,17 +112,17 @@ struct htif_config final {
     uint64_t fromhost{FROMHOST_INIT}; ///< Value of fromhost CSR
     uint64_t tohost{TOHOST_INIT};     ///< Value of tohost CSR
     bool console_getchar{false};      ///< Make console getchar available?
-    bool yield_manual{false};         ///< Make yield manual available?
-    bool yield_automatic{false};      ///< Make yield automatic available?
+    bool yield_manual{true};          ///< Make yield manual available?
+    bool yield_automatic{true};       ///< Make yield automatic available?
 };
 
 /// \brief Rollup configuration
-struct rollup_config {                    // NOLINT(bugprone-exception-escape)
-    memory_range_config rx_buffer{};      ///< RX buffer
-    memory_range_config tx_buffer{};      ///< TX buffer
-    memory_range_config input_metadata{}; ///< Buffer for input metadata
-    memory_range_config voucher_hashes{}; ///< Buffer for the voucher hash array
-    memory_range_config notice_hashes{};  ///< Buffer for the notice hash array
+struct rollup_config {                                                          // NOLINT(bugprone-exception-escape)
+    memory_range_config rx_buffer{.start = 0x60000000, .length = 2 << 20};      ///< RX buffer
```

Comment:
It compiles, but I am getting these warnings:
```c++
machine-config.h:121:35: warning: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Wc++20-extensions]
  121 |     memory_range_config rx_buffer{.start = 0x60000000, .length = 2 << 20};      ///< RX buffer
      |                                   ^
machine-config.h:121:56: warning: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Wc++20-extensions]
  121 |     memory_range_config rx_buffer{.start = 0x60000000, .length = 2 << 20};      ///< RX buffer
      |                                                        ^
```

Using `memory_range_config rx_buffer{0x60000000, 2 << 20};` should fix.

Reply:
I used it in LambaDEX, but I selected c++20. Victor must have seen it. Shall we make the switch? This is really cool. :)